.github/renovate.json5

```diff
@@ -141,6 +141,7 @@
       ],
       "datasourceTemplate": "github-tags",
       "depNameTemplate": "github/codeql-action",
+      "extractVersionTemplate": "^codeql-bundle-(?<version>.*)$"
```

Comment:
@phisco The `github/codeql-action` repo has two forms of tag:

* vX.Y.Z
* codeql-bundle-vX.Y.Z

I want to make sure Renovate only considers the latter. Do you think this will work?